internal/support/notifications/rest_notification.go

```diff
@@ -265,16 +382,12 @@ func notificationByStartEndHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	n, err := dbClient.GetNotificationsByStartEnd(start, end, limitNum)
+	n, err := getNotificationsByStartEnd(start, end, limitNum)
 	if err != nil {
-		if err == db.ErrNotFound {
```

Comment:
Why did you remove this `ErrNotFound` check?

Reply:
This function does not return a ErrNotFound, it returns an empty json array when it doesn't find anything